cpp/harvest/nacjd.cc

```diff
@@ -43,11 +43,18 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "marc_title_in_file new_marc_title_out_file [count_of_rows]\n"
+        "marc_title_in_file new_marc_title_out_file [mode] [mode_params]\n"
```

Comment:
This should only be `[mode] [mode_params]`. The other params are totally depending on the mode. E.g. if we download a single ID, we do not need marc_title_in_file and new_marc_title_out_file.